### PR TITLE
Use mv instead cp to save space

### PR DESCRIPTION
**Pull request by spinolacastro**
@bparees this have been used on 5.5 and 5.6 tags without issues, PTAL.

fixes https://github.com/sclorg/s2i-php-container/issues/106

### CODE REVIEW

**Comment by bparees** [comment]
[test]

File: 5.5/s2i/bin/assemble [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 set -e
 
+shopt -s globstar
 echo "---> Installing application source..."
-cp -Rf /tmp/src/. ./
+mv /tmp/src/* ./
```
Comment (bparees) [inline 5.5/s2i/bin/assemble]: i think you need dotglob or you're going to miss dot files.

Comment (spinolacastro) [inline 5.5/s2i/bin/assemble]: Isn't  `shopt -s globstar` supposed to do that?

Comment (bparees) [inline 5.5/s2i/bin/assemble]: not based on my reading and not based on my testing:
```
ls -a
.  ..  .bar  .foo  foo2  .s2i
```

```
$ shopt -s globstar
$ ls *
foo2
```

```
$ shopt -s dotglob
$ ls *
.foo  foo2

.bar:
foo2

.s2i:
bin
```

Comment (spinolacastro) [inline 5.5/s2i/bin/assemble]: Sorry, my bad.

File: 5.5/s2i/bin/assemble [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 set -e
 
+shopt -s globstar dotglob
```
Comment (bparees) [inline 5.5/s2i/bin/assemble]: globstar isn't needed/doing anything
http://www.linuxjournal.com/content/globstar-new-bash-globbing-option

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
lgtm

**Pull request merged**